OPERATING.md

```diff
@@ -20,6 +20,45 @@ yyyy-mm-dd HH:mm:ss.SSS Level=DEBUG Logger=u.n.a.p.t.s.BundleMapperService Conve
 
 ## Timeout functionality
 
+The Adaptor conforms to the GP2GP large messaging specification (NPFIT-PC-BLOD-0170.03 v1.1)
+by timing out in-progress transfers (section 2.3.6). This ensures transfers are ended gracefully
+in the scenario a GP2GP message has not been received.
+
+The timeout 
+datetime is calculated using the following formula and the *Persist Duration* (the minium time a message is persisted by spine) of the expected GP2GP messages.
```

Comment:
This is a little confusing, the formula includes the persist duration as far as I can tell.